.github/workflows/update-helm-repo.yaml

```diff
@@ -234,7 +233,9 @@ jobs:
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
-          password: ${{ env.AUTHTOKEN }}
+          # GitHub App installation token cannot push package to the org's package registry;
+          # using GITHUB_TOKEN here instead to avoid using PAT.
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
what was the reason for using `AUTHTOKEN` at all in this workflow? isn't it that we want to inject a token from another repo which might have different permissions from the `GITHUB_TOKEN` that the helm-charts repo gets?

Reply:
`GITHUB_TOKEN` doesn't let you push content cross-repo (i.e. from `mimir` to `helm-charts`), thus the options are either use a PAT or GitHub app installation. But packages live in the org — in `ghcr.io/grafana` — and only PAT or `GITHUB_TOKEN` can push there.

Reply:
ok, then i'm with you